cfgov/cfgov/settings/production.py

```diff
@@ -88,8 +88,9 @@
 
 CACHES = {
     'default': {
-        'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache',
-        'LOCATION': '/tmp/eregs_cache',
+        'BACKEND': 'django.core.cache.backends.db.DatabaseCache',
+        'LOCATION': 'cfgov_default_cache',
+        'TIMEOUT': None,
```

Comment:
To test this, you can copy this config:

```python
CACHES = {
    'default': {
        'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache',
        'LOCATION': '/tmp/eregs_cache',
        'BACKEND': 'django.core.cache.backends.db.DatabaseCache',
        'LOCATION': 'cfgov_default_cache',
        'TIMEOUT': None,
    }
}
```

Into `cfgov/cfgov/settings/local.py`, and then run

```
./cfgov/manage.py createcachetable
```

Then you should be able to cache in the shell:

```python

```
from django.core.cache import cache
cache.set('my_key', 'hello, world!', 10)
cache.get('my_key')
```

You should get `hello, world!` back.

Then wait 10 seconds and try again:

```
cache.get('my_key')
```

And you should get nothing.

Reply:
Thanks for the steps to test.